[OSDOCS#10055]: Document manual rotation of etcd signer certificates (#77406)
openshift/openshift-docs: modules/etcd-cert-alerts-metrics-signer.adoc

// Module included in the following assemblies:
//
// * security/certificate_types_descriptions/etcd-certificates.adoc

:_mod-docs-content-type: CONCEPT    
[id="etcd-cert-alerts-metrics-signer_{context}"]
== etcd certificate rotation alerts and metrics signer certificates

Two alert types inform users about pending `etcd` certificate expiration:

> [horizontal]

Reviewer: Do you know for sure if our tooling handles this well (looks good on the preview, but I haven't seen it before so wonder about the Portal :sweat_smile:)

Author: @jeana-redhat I did a quick look at it, but couldn't find an example online. I decided that because it was documented in our doc references, that it would not be an issue. :-) I will make a note to verify this list after it hits the portal! Thanks for that input!

Reviewer 2: Oooh.  I do not like that look at all! :grinning:

![image](https://github.com/openshift/openshift-docs/assets/25209155/393387f1-e2ae-4365-a5cf-96b992901b1e)

Reviewer 3: Please validate that this looks ok on docs.redhat after GA.

`etcdSignerCAExpirationWarning`:: Occurs 730 days until the signer expires.
`etcdSignerCAExpirationCritical`:: Occurs 365 days until the signer expires.

> You can rotate the certificate:
> 
> * When you receive an expiration alert
> * When the private key is leaked

Reviewer: ```suggestion
You can rotate the certificate for the following reasons:

* You receive an expiration alert
* The private key is leaked
```

> NOTE: When a private key is leaked, you must rotate all of the certificates.

Reviewer: Honestly wonder if this is more important than a note :thinking: 

```suggestion
[NOTE]
====
When a private key is leaked, you must rotate all of the certificates.
====
```

Author: @jeana-redhat I can't find it now where I saw this construct for an admonition documented (but I will!). However, I will make this change and also change it to 'IMPORTANT', because I agree that it is more impacting than having it as a note.

// Module included in the following assemblies:
//
// * security/certificate_types_descriptions/etcd-certificates.adoc

:_mod-docs-content-type: CONCEPT    
[id="etcd-cert-alerts-metrics-signer_{context}"]

> == etcd certificate rotation alerts and metrics signer certificates

Reviewer 3: ```suggestion
= etcd certificate rotation alerts and metrics signer certificates
```
The first heading in a module is always an H1. Adjust the leveloffset in the assembly, if necessary.

Two alert types inform users about pending `etcd` certificate expiration:
[horizontal]
`etcdSignerCAExpirationWarning`:: Occurs 730 days until the signer expires.
`etcdSignerCAExpirationCritical`:: Occurs 365 days until the signer expires.

You can rotate the certificate for the following reasons:

> * You receive an expiration alert
> * The private key is leaked

Reviewer 2: Per ISG: If the list items comprise only complete sentences, include a period after each sentence.
```suggestion
* You receive an expiration alert.
* The private key is leaked.
```

[IMPORTANT]
====

> When a private key is leaked, you must rotate all of the certificates.

Reviewer 2: I assume a standard user would know when a key is leaked and what that means....
